README.md

```diff
@@ -1,21 +1,105 @@
-The **μAgents** (micro-Agents) project is a fast and lightweight framework that makes it easy to build agents for all kinds of decentralised use cases.
+# uAgents: AI Agent Framework
+[![Official Website](https://img.shields.io/badge/Official%20Website-fetch.ai-blue?style=flat&logo=world&logoColor=white)](https://fetch.ai) [![Unit Tests](https://img.shields.io/github/actions/workflow/status/Fetchai/uAgents/ci.yml?label=unit%20tests)](https://github.com/Fetchai/uAgents/actions/workflows/ci.yml) [![GitHub Repo stars](https://img.shields.io/github/stars/Fetchai/uAgents?style=social)](https://github.com/Fetchai/uAgents/stargazers) [![Twitter Follow](https://img.shields.io/twitter/follow/fetch_ai?style=social)](https://twitter.com/fetch_ai)
+
+uAgents is a library developed by Fetch.ai that allows for creating autonomous AI agents in Python. With simple and expressive decorators, you can have an agent that performs various tasks on a schedule or takes action on various events.
+
+## 🚀 Features
+
+- 🤖 **Easy creation and management**: Create any type of agent you can think of and put into code.
+- 🔗 **Connected**: On startup, each agent automatically joins the fast growing network of uAgents by registering on the Almanac, a smart contract deployed on the Fetch.ai blockchain.
+- 🔒 **Secure**: uAgent messages and wallets are cryptographically secured, so their identities and assets are protected.
+
+## ⚡ Quickstart
+
+### Installation
+Get started with uAgents by installing it for Python 3.8, 3.9, or 3.10:
+
+    poetry install 
+    poetry shell
+
+### Running a Demo
+
+#### Creating an Agent 
+Build your first μAgent using the following script:
```

Comment:
I think it was decided to use `uAgents` as the common standard name for code and docs.
```suggestion
Build your first uAgent using the following script:
```